metashare/api/models.py

```diff
@@ -168,19 +179,21 @@ def __str__(self):
     class Meta:
         verbose_name_plural = "repositories"
         ordering = ("name",)
+        unique_together = (("repo_owner", "repo_name"),)
 
 
 class GitHubRepository(mixins.HashIdMixin, models.Model):
-    url = models.URLField()
     user = models.ForeignKey(
         User, on_delete=models.CASCADE, related_name="repositories"
     )
+    repo_id = models.IntegerField(unique=True)
+    repo_url = models.URLField()
```

Comment:
The only reasons to store this IMO are 1) it's trivial, and 2) for ease of DB sleuthing and admin browsing.